### DISCUSSION TITLE
Hide Controls for props not pass through Args

### DISCUSSION THREAD

**Discussion by joshjeong** [storybookjs/storybook#11324] [Ideas]
```
export const Primary = (args) => <Button {...args} />;
Primary.args = { label: 'this is a very long string', background: '#ff0' };
```
Button also has an onClick prop but since it isn't passed in args, it would be great if there was no input field in the Controls table or a way to disable.

<img width="616" alt="Screen Shot 2020-06-25 at 11 26 39 PM" src="https://user-images.githubusercontent.com/6474199/85827553-5aa5f300-b73b-11ea-9616-c95aeb136faf.png">

**Comment by shilman** [comment]
If the `argType` of onClick is properly inferred (either from `propTypes` or TypeScript), it should be a `function` and there should be no control. Since the type is displaying properly I'd also expect there to be no control. Do you have a repro?

**Comment by joshjeong** [comment]
Thank you! propTypes and this pr are what I was looking for! https://github.com/storybookjs/storybook/pull/11388
